packages/@guardian/source-foundations/src/typography/types.ts

```diff
@@ -48,12 +48,51 @@ type Categories = {
 
 export type Category = keyof Categories;
 
-export type AvailableFontsMapping = {
-	[cat in Category]: {
-		[fontWeight in FontWeight]?: FontWeightDefinition;
-	};
+export type AvailableFontWeights = {
+	[cat in Category]?: { [weight in FontWeight]?: boolean };
 };
 
+export type ItalicsAvailableForFontWeight = {
+	[cat in Category]?: { [weight in FontWeight]?: boolean };
+};
+
+export type TypographyOptions = {
```

Comment:
Again i wonder if using `spec` rather than `options` makes mores sense? I'm thinking of `options` in terms of what the consumer uses to change the spec rather than the spec itself

Reply:
agreed - what d'you think about `TypographyConfiguration`?

Reply:
Yeah sounds good to me.